shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -436,4 +449,18 @@ public Options getOptions() {
   public int numArgs() {
     return 0;
   }
+
+  /**
+   * Determine is force is set as an option or user enters (y | yes) on the shell prompt.
+   *
+   * @return true if force is set as opt or y | yes entered at command line.
+   */
+  private boolean forceSet(final Shell shellState, final CommandLine cl, final String prompt) {
+    if (cl.hasOption(forceOpt)) {
+      return true;
+    }
+    shellState.getWriter().flush();
+    String line = shellState.getReader().readLine(prompt + " (yes|no)? ");
+    return line != null && (line.equalsIgnoreCase("y") || line.equalsIgnoreCase("yes"));
```

Comment:
This prompt feels very custom. I suspect there are other places in the shell where we have a similar prompt. It'd be good to maintain consistency across the shell for yes/no prompts. I suggest putting this in the Shell class, or moving any existing similar method into the Shell class, and calling it there, as in:

```java
    if (cl.hasOption(forceOpt) || shellState.yesNo(prompt)) {
      // ...
    }
```

Reply:
I would prefer to do that as a separate PR. The code was copied / modified from delete table.  I will do this as a follow-on,

Reply:
Addressed with PR #3684